R/json.tabular.to.data.frame.R

```diff
@@ -6,20 +6,10 @@
 # of patent rights can be found in the PATENTS file in the same directory.
 
 #' @include dbDataType.R
+#' @useDynLib RPresto
+#' @importFrom Rcpp sourceCpp
```

Comment:
Seems so so that Roxygen can generate NAMESPACES properly.
https://github.com/hadley/devtools/blob/01ab167ca8663562c30a2524d3f2907b2c2ba10d/R/infrastructure.R#L125